src/wilcoxon.jl

```diff
@@ -129,10 +129,13 @@ function pvalue(x::ExactSignedRankTest; tail=:both)
     end
 end
 
+ci(x::ExactSignedRankTest, alpha::Float64=0.05; tail=:both) = calculate_ci(x.vals, alpha; tail=tail)
+
 
 ## APPROXIMATE SIGNED RANK TEST
 
 immutable ApproximateSignedRankTest{T<:Real} <: HypothesisTest
+    vals::Vector{T}         # original values
```

Comment:
let me know if anybody has any objections to this but I need the original data to compute the confidence intervals (let me know if you know of any alternative ways to compute the CIs without having to have access to the original data)